agents_infra/install

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+# Setup build environment
+apt update && apt install -y ruby ruby-dev build-essential ansible rsync sudo
+gem install --no-document dotenv:2.8.1 fpm:1.15.1
```

Comment:
It may help to put these into a `Gemfile` and use `bundler` for easier upgrades later.